_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+* Headers, indicated by `#`. `#` corresponds to an `H1` header, `##` and `H2` header, and so on.
+* Bullet-points, which can be created using `*` or `-`.
+* Checkboxes, which appear as `[ ]`. You can mark a checkbox as completed by adding an "X", like this: `[X]`. The "X" can be uppercase or lowercase.
+* Bolding text, using two asterisks on each side of the text to be bolded, like this: **this text is bold**.
+* Italicizing text, using a single asterisk on each side of the text to be italicized: *this text is italicized*.
+
+> New to Markdown? Here's a great [Markdown Guide](https://www.markdownguide.org/basic-syntax/) that covers all of the basic syntax.
+
+## <i class="fa-solid fa-map"></i> Step-by-step guide to completing the pyOpenSci software review issue template
+
+When completing your package submission, it's best to submit the issue all at once, rather than completing it over the course of several sessions. We recommend reading through the issue once or twice to see what information you'll need, which includes:
+
+* The scope of your package (see categories)
+* Your target audience
+* Scientific applications of your package
+* Other packages that accomplish the same thing (if any), and how yours is different
+
+> Remember: keep the formatting of the issue template exactly as it is! It's OK to leave sections that don't apply to you blank, and there's no need to add any additional formatting (bolding, italicizing, etc.). Any of these types of changes make it more difficult for our editorial team to process your package submission.
+
+### Package metadata
+
+* **Add a title:** keep your title simple. Oftentimes your package name and the word submission is enough, such as "My Amazing Python package submission."
+* **Add a description** in this section you'll want to fill out the Submitting Author, All current maintainers, Package Name, One-Line Description of Package, Repository Link, and Version submitted sections. You should *not* fill out the EiC, Editor, Reviewer 1, Reviewer 2, Archive, JOSS DOI, Version accepted, or Date accepted sections. Instead, leave these as is, with "TBD."
+
+### Code of Conduct & Commitment to Maintain Package
+
+In order for your package to progress through the pyOpenSci review process, you'll need to check both of the [Code of Conduct](https://www.pyopensci.org/handbook/CODE_OF_CONDUCT.html) agreement as well as the [maintenance agreement](https://www.pyopensci.org/software-peer-review/our-process/policies.html#after-acceptance-package-ownership-and-maintenance) outlined in the [pyOpenSci Policies Guidelines](https://www.pyopensci.org/software-peer-review/about/intro.html). You can indicate your agreement in one of two ways:
```

Comment:
This feels like we are telling them something they will know but not the context.This sectino would be more useful if it was a statement along the lines of:

all participants of our peer review process need to follow the pyOpenSci CoC... 

then add a short sentence about why this is important to us and our values 

For the long term maintenance - it would be important to reference here why we have that box - we want scientists to be able to find vetted maintained tools, etc. So rather than just telling them what to check, i think a bit of information on why this aligns with our organization and program values is important. it can be short! but it should be here and it's more important than the labels and markdown pieces.

when we write we should always be aligning our values to content delivery and approach.